### PR TITLE
enable incremental repair during global routing

### PR DESCRIPTION
**Pull request by eder-matheus**
* Update gf12/ariane timing constraints
* Update a few designs metrics related to instance area
* Update global route resource adjustments for sky130hd and sky130hs

### CODE REVIEW

File: flow/scripts/global_route.tcl [hunk 1]
```diff
@@ -24,18 +24,20 @@ if {[info exist env(FASTROUTE_TCL)]} {
...
-check_placement -verbose
 
 set_propagated_clock [all_clocks]
 estimate_parasitics -global_routing
 
+if {[info exist env(DONT_USE_CELLS)]} {
+  set_dont_use $::env(DONT_USE_CELLS)
+}
+
 if { [info exists ::env(RECOVER_POWER)] } {
```
Comment (maliberty) [inline flow/scripts/global_route.tcl]: recover_power should happen after incremental repair

File: flow/scripts/global_route.tcl [hunk 2]
```diff
@@ -49,13 +51,48 @@ if { [info exists ::env(RECOVER_POWER)] } {
...
 
 puts "\n=========================================================================="
 puts "check_antennas"
 puts "--------------------------------------------------------------------------"
+
+repair_antennas -iterations 5
+check_placement -verbose
 check_antennas -report_file $env(REPORTS_DIR)/antenna.log -report_violating_nets
 
+set_propagated_clock [all_clocks]
```
Comment (maliberty) [inline flow/scripts/global_route.tcl]: We already did this in line 36

**Review by maliberty — CHANGES_REQUESTED** [review]

File: flow/scripts/global_route.tcl (hunk 1 above)
Comment (eder-matheus) [reply flow/scripts/global_route.tcl]: Fixed

File: flow/scripts/global_route.tcl (hunk 2 above)
Comment (eder-matheus) [reply flow/scripts/global_route.tcl]: Fixed. Are there other comments on this PR? Just to make sure everything is correct here before starting the final CI run.

Comment (maliberty) [reply flow/scripts/global_route.tcl]: go ahead

File: flow/designs/gf12/ariane/constraint.sdc [hunk 3]
```diff
@@ -1,496 +1,496 @@
...
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[7]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[6]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[5]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[4]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[3]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[2]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[1]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[0]}]
+create_clock [get_ports clk_i]  -name core_clock  -period 3400  -waveform {0 850}
+set_input_delay -clock core_clock  1000  [get_ports clk_i]
```
Comment (rovinski) [inline flow/designs/gf12/ariane/constraint.sdc]: Does not make sense to add a delay to the clock port

File: flow/designs/gf12/ariane/constraint_hier.sdc [hunk 4]
```diff
@@ -1,496 +1,496 @@
...
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[8]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[7]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[6]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[5]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[4]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[3]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[2]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[1]}]
-set_output_delay -clock core_clock  0  [get_ports {axi_req_o[0]}]
+set_input_delay -clock core_clock  1500  [get_ports clk_i]
```
Comment (rovinski) [inline flow/designs/gf12/ariane/constraint_hier.sdc]: Same

**Comment by eder-matheus** [comment]
@maliberty Public ci is green. Private ci is almost clean. Only one design has a failing metric. I temporarily disabled incremental repair for it and will investigate once this PR is merged.

**Review by maliberty — APPROVED** [review]

**Comment by eder-matheus** [comment]
@maliberty This is ready to merge now.

**Pull request merged**